spec/features/service_drafts/service_draft_form_navigation_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe 'Service Draft form navigation' do
+  let(:form_titles) { ['Service Information', 'Options', 'Coverage'] }
```

Comment:
should we have this include all forms?

Reply:
yep, those got added in after I had this test implemented